<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSGoodsSmallCategoryWidget</class>
 <widget class="QWidget" name="BCSGoodsSmallCategoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>102</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="OrderCategorySmallListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView{
border:none;
background-color: rgb(81, 80, 80);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="lastPageButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/orderGoodsImage/changePageUp.png);
}

QPushButton:pressed{
border-image: url(:/orderGoodsImage/changePageUp_press.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextPageButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/orderGoodsImage/changePage.png);
}

QPushButton:pressed{
border-image: url(:/orderGoodsImage/changePage_press.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderCategorySmallListView</class>
   <extends>QListView</extends>
   <header>OrderCategorySmallListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
